app/styles/modules/_global_header.scss

```diff
@@ -2,7 +2,7 @@
   @include color-scheme($white, $middle-grey);
   position: fixed;
   top: 0;
-  z-index: 100;
+  z-index: 10;
```

Comment:
Yeah good idea. I'm hopeful we won't need much beyond what we're using now, but that would definitely help keep things sane.